### ISSUE TITLE
yakbak('http://host.com/subdir') does not work

### ISSUE THREAD

**Issue by mwolson** [flickr/yakbak#16]
In this case:

```js
yakbak('http://host.com/subdir', {dirname});
```

It seems like yakbak drops the '/subdir' part when it proxies requests and only sends 'http://host.com/'.

**Comment by commenthol** [comment]
Proxying paths is not as trivial as proxying hosts. If you need path proxying you could do this with [proxy-middleware](https://www.npmjs.com/package/proxy-middleware) e.g. 

```js
const app = express()

app.use('/subdir', yakbak('http://host.com', {dirname}))
app.use('/', proxyMiddleware('http://localhost:3000/subdir'))

app.listen(3000)
```

**Comment by jeremyruppel** [comment]
Yeah, yakbak isn't meant to do any sort of routing, it's a simple proxy from one host to another. Are you using yakbak with express by chance? Internally we use `req.url` which [express rewrites when you do `app.use()`][1].

[1]: http://expressjs.com/en/4x/api.html#req.originalUrl

**Comment by mwolson** [comment]
I'm using express. But I actually want the req.url rewriting since I'm
proxying multiple services on different routes (namespaced by scenario,
then service, then subservice) each with different tape directories on one
express server. The problem is that one of those services has some leading
path fragments which are present in dev but not production. So I'd like to
have yakbak handle those leading fragments to simplify the rest of my setup.

**Comment by jeremyruppel** [comment]
Hm, I'm not sure I'm quite understanding the setup here. Could you gist an example when you have a chance?
